test/lib/mayaUsd/render/mayaToHydra/testTransforms.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020 Luma Pictures
+# Copyright 2023 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import maya.cmds as cmds
+import maya.mel
+
+import fixturesUtils
+import mtohUtils
+import unittest
+
+class TestTransforms(mtohUtils.MtohTestCase):
+    _file = __file__
+
+    IMAGEDIFF_FAIL_THRESHOLD = 0.01
+    IMAGEDIFF_FAIL_PERCENT = 0.1
```

Comment:
I think we use 2 for the percentage error , but this may fail if you're checking the selection highlight of a node for example. But maybe 0.1 is a bit low. We'll see from the preflight.